conventions/documentation/alerting.md

```diff
@@ -0,0 +1,72 @@
+---
+parent: Conventions
+---
+
+# Alerting
+
+## Inline Alerts
+Inline Alerts can be used throughout the console to notify users in context that something is happening, happened, or needs their attention.
+
+### Inline Alerts in Details Views
+
+#### Above navigation tabs
+Inline alerts can appear above the navigation tabs when there is an important piece of information that applies to the entire resource, and the user should be made aware of regardless what tab they are looking at.
+
+![inline alert details above tabs](../images/alerting-inline-details-above.png)
+- Generally, these should be warning or error level alerts.
+- The alert should appear immediately above the navigation tabs.
+
+
+#### Below navigation tabs
+Inline alerts can appear at the top of an individual navigation tab when there is a particular piece of information that applies to that one tab, and is not severe enough to surface at all times in the resource.
+
+![inline alert details below tabs](../images/alerting-inline-details-below.png)
+
+![inline alert details below tabs 2](../images/alerting-inline-details-below2.png)
+
+- Generally, these should be info or neutral alerts.
+- If a warning or error level alert is not severe enough to surface at all times in the resource, it can also be surfaced only in the tab it relates to.
```

Comment:
If I'm reading this correct it seems to conflict with the example above where we show a warning alert in the tab content area

Reply:
I attempted to reword this sentence to make it clear that warning/error inline alerts can appear in a tab if it's determined they aren't severe enough to always show (but am still open to feedback on this.)